PopPUNK/plot.py

```diff
@@ -624,8 +624,7 @@ def writeClusterCsv(outfile, nodeNames, nodeLabels, clustering,
                                  col + "(length of " + this_col_items + ")\n")
         sys.exit(1)
 
-
-def buildRapidNJ(rapidnj, refList, coreMat, outPrefix, tree_filename, threads = 1):
+def buildRapidNJ(rapidnj, refList, coreMat, outPrefix, threads = 1):
```

Comment:
I think it would make sense to move all of the tree functions in `plot.py` into their own new module file

Reply:
Done in 8e50745.